### ISSUE TITLE
Contact via email

### ISSUE THREAD

**Issue by buhtz** [axfelix/axfelix.github.io#1]
Dear Alex,
my apologize contacting you this way. But I can't find any other "channel" to reach you.

Is there a way to get in contact? Related to my role as member of the maintenance team of [Back In Time](https://github.com/bit-team/backintime) I have some questions about an article you wrote on DigitalOcean.

You can find my email contact in the meta data of my projects; for example [here](https://codeberg.org/buhtz/buhtzology/src/commit/d9692fea39607907d35e9271107201ad3d020fd6/pyproject.toml#L29).

Thanks in advance,
Christian

**Comment by fernandodiacenco** [comment]
Hi I'm in the same boat, trying my best to find Alex contact info to ask about a dante config.
@axfelix Where can I ping you? Thanks.

**Comment by axfelix** [comment]
Hi folks! I no longer work for Digitalocean or have the ability to edit those tutorials, so you'll have to consider them "as-is." What can I do for you?

**Comment by fernandodiacenco** [comment]
Hey thanks for answering. In my case I'm trying dante server BUT with transparent proxy using TPROXY on nftables, is this possible? I managed to get dante working, and managed to get a supposedly working config on nftables, buts it's not tproxying at all so I started to think that maybe dante can't do it.

**Comment by axfelix** [comment]
Yeah, my assumption would be no -- at that point I'd check with the Dante maintainers.

**Comment by fernandodiacenco** [comment]
Thank you, have a nice weekend.
